mass_mailing_custom_unsubscribe_event/models/event_registration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    opt_out = fields.Boolean(
+        string='Opt-Out',
```

Comment:
I think this field should be tracked.

```suggestion
        string='Opt-Out',
        track_visibility="onchange",
```

Reply:
As we already have the unsubscription log, I see this as redundant and less "trackable" than the other record.